recipe/meta.yaml

```diff
@@ -29,7 +29,9 @@ test:
 
 about:
   home: https://github.com/blink1073/tifffile
-  license:  BSD License
+  license: BSD 3-Clause
```

Comment:
Just FYI, this results in an additional change to the Readme after re-rendering. Not that anything needs to be done about it at this time. Simply providing information.